wsclient/src/main/java/org/hpccsystems/ws/client/wrappers/wsworkunits/ECLQueryWrapper.java

```diff
@@ -0,0 +1,175 @@
+package org.hpccsystems.ws.client.wrappers.wsworkunits;
+
+public class ECLQueryWrapper
+{
+    private String text;
+
+    private String cpp;
+
+    private String resTxt;
+
+    private String dll;
+
+    private String thorLog;
+
+    private String queryMainDefinition;
+
+    public ECLQueryWrapper() {}
+    
+    public ECLQueryWrapper(org.hpccsystems.ws.client.gen.wsworkunits.v1_73.ECLQuery raw) {
+        if (raw==null) {
+            return;
+        }
+        this.text=raw.getText();
+        this.cpp=raw.getCpp();
+        this.dll=raw.getDll();
+        this.queryMainDefinition=raw.getQueryMainDefinition();
+        this.thorLog=raw.getThorLog();
+        this.resTxt=raw.getResTxt();
+    }
+   
+    public ECLQueryWrapper(org.hpccsystems.ws.client.gen.wsworkunits.v1_69.ECLQuery raw) {
+        if (raw==null) {
+            return;
+        }
+        this.text=raw.getText();
+        this.cpp=raw.getCpp();
+        this.dll=raw.getDll();
+        this.queryMainDefinition=raw.getQueryMainDefinition();
+        this.thorLog=raw.getThorLog();
+        this.resTxt=raw.getResTxt();
+    }
+
+    public org.hpccsystems.ws.client.gen.wsworkunits.v1_73.ECLQuery getRaw() {
+        return getRawVersion73();
+    }
+
+    public org.hpccsystems.ws.client.gen.wsworkunits.v1_73.ECLQuery getRawVersion73() {

```

Comment:
in this case the 73 seems to map to the wsworkunits interface version in getRawVersion6 I thought  it was mentioned the 6 stood for the HPCC Major version.
It prob makes sense to make these HPCC major version specific so, getRawHPCC7xVersion ?